conan/test_package/CMakeLists.txt

```diff
@@ -1,13 +1,44 @@
 cmake_minimum_required(VERSION 3.1)
 project(PackageTest CXX)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/cmake)
-
 find_package(userver REQUIRED)
 
+add_library(${PROJECT_NAME}_objs OBJECT
+    hello.hpp
+    hello.cpp
+)
+target_link_libraries(${PROJECT_NAME}_objs PUBLIC userver-core)
```

Comment:
Should we perhaps follow best practices and export `userver::core` etc.?

Reply:
I just add the same targets as it declared in cmakelists. It can be changed to anything else, how would you prefer